clustering/jgroups/extension/src/test/resources/org/jboss/as/clustering/jgroups/subsystem/subsystem-jgroups-transform-reject.xml

```diff
@@ -75,12 +76,7 @@
             </relay>
         </stack>
         <stack name="minimal">
-            <transport type="UDP">
-                <default-thread-pool min-threads="11"/>
-                <internal-thread-pool queue-length="22"/>
-                <oob-thread-pool keepalive-time="34"/>
-                <timer-thread-pool max-threads="43"/>
```

Comment:
WFLY-6766 makes validation of the rejection of these resources impossible.  Consequently, I need to either remove these items from the configuration being tested, or ignore the whole test.